Automattic/jetpack
# Publicize: Configure Publicize only if the Publicize Module is active.

We are loading in Publicize in Jetpack, even if the Publicize module is inactive. We need to load Publicize only if the module is active. 



To fix p1654201798873139-slack-CDD9LQRSN and https://github.com/Automattic/jetpack/pull/23871

#### Changes proposed in this Pull Request:

* Configure Publicize in Jetpack only if the Publicize Module is active. 

#### Jetpack product discussion


https://github.com/Automattic/jetpack/pull/23871#issuecomment-1140746502

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






- Install Jetpack and make sure Publicizing works correctly
- Deactivate the Publicize module and make sure all the Publicize elements get hidden on the block editor.
- Activate the Publicize module again and make sure you're able to Publicize correctly.
- Deactivate the Publicize module and deactivate Jetpack, activate the Jetpack Social Plugin and make sure you're able to - Publicize via the Classic Editor.
